cc/N50.挑选代表/挑选代表.cc

```diff
@@ -0,0 +1,33 @@
+#include<bits/stdc++.h>
+using namespace std;
+int main(){
+    int n;
+    cin >> n;
+    map<int,int> m;
+    int a,b;
+    for(int i=0;i<n;i++){
+        cin >> a >> b;
+        m.insert({b,a});
+    }
+    map<int,int>::iterator p;
+    p=m.begin();
```

Comment:
you can try auto.
like `map<int,int>::iterator p = m.begin` ==> `auto p = m.begin()`